src/tiffimage_int.cpp

```diff
@@ -1695,10 +1695,8 @@ namespace Exiv2 {
             TiffFinder finder(0x00fe, imageGroups[i]);
             pSourceDir->accept(finder);
             TiffEntryBase* te = dynamic_cast<TiffEntryBase*>(finder.result());
-            if (   te
-                && te->pValue()->typeId() == unsignedLong
-                && te->pValue()->count() == 1
-                && (te->pValue()->toLong() & 1) == 0) {
+            const Value* pV = te != NULL ? te->pValue() : NULL;
```

Comment:
`nullptr` please.

Reply:
It cannot be in 0.27, where we are using C++98 😉

Reply:
Wrong branch I am, sorry.